src/EOFTestsFiller/EIP4200/validInvalidFiller.yml

```diff
@@ -0,0 +1,1217 @@
+      expectException:
+        "Shanghai" : "EOF_InvalidCode"
+
+    # EOF1V4200_0013 (Valid) EOF with RJUMPV table size 255 (Target 256)
+    # Code: PUSH2(256) RJUMPV(random 255 offsets) NOP * 255 PUSH1(1) PUSH1(1) SSTORE STOP
+    - data: |
+       :label EOF1V4200_0013 :raw 0xef0001010004020001030803000000000000026101005eff006700c600690073005100ff004a00ec002900cd00ba00ab00f200fb00e30046007c00c2005400f8001b00e800e7008d0076005a002e00630033009f00c9009a00660032000d00b70031005800a3005a0025005d00050017005800e9005e00d400ab00b200cd00c6009b00b400540011000e0082007400410021003d00dc0087007000e9003e00a1004100e100fc0067003e0001007e009700ea00dc006b0096008f0038005c002a00ec00b0003b00fb003200af003c005400ec001800db005c0002001a00fe004300fb00fa00aa003a00fb002900d100e60005003c007c0094007500d800be0061008900f9005c00bb00a80099000f009500b100eb00f100b3000500ef00f7000000e900a1003a00e500ca000b00cb00d000480047006400bd001f0023001e00a8001c007b006400c500140073005a00c5005e004b00790063003b0070006400240011009e000900dc00aa00d400ac00f2001b001000af003b003300cd00e30050004800470015005c00bb006f0022001900ba009b007d00f5000b00e1001a001c007f002300f8002900f800a4001b001300b500ca004e00e800980032003800e00079004d003d003400bc005f004e007700fa00cb006c000500ac00860021002b00aa001a005500a200be007000b50073003b0004005c00d30036009400b300af00e200f000e4009e004f00320015004900fd008200ff5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b600160015500
+      expectException:
+        "Shanghai" : "EOF_InvalidCode"
+
+    # EOF1I4200_0001 (Invalid) EOF code containing truncated RJUMP
+    # Code: RJUMP
+    - data: |
+       :label EOF1I4200_0001 :raw 0xef0001010004020001000103000000000000005c
+      expectException:
+        "Shanghai" : "EOF_InvalidCode"
+        ">=Cancun" : "EOF_TruncatedImmediate"
```

Comment:
Why is there difference between forks?

All RJUMP-related changes supposedly target the single fork.